resolver/resolver.go

```diff
@@ -1701,6 +1701,8 @@ func (r *Resolver) resolveMessageArgument(ctx *context, files []*File) {
 	if graph == nil {
 		return
 	}
+
+	serviceMsgMap := make(map[string]map[string]*Message)
```

Comment:
Another option may be to change [the Service.Messages and Service.MessageArgs](https://github.com/mercari/grpc-federation/blob/7b0c2fe6328431246a9cc29907e119606132bd4e/resolver/types.go#L40C14-L41) from slice to map, but then sorting it is kind of complicated, so I decided to use the serviceMsgMap instead.